challenge_restaurant/src/challenge_restaurant/take_orders.py

```diff
@@ -135,6 +136,10 @@ def execute(self, userdata=None):
 
         self._robot.speech.speak(sentence)
 
+        if "water" in self._orders:
+            water_sentence = "I am too weak to carry the water, please bring that yourself. And please make me stronger."
+            self._robot.speech.speak(water_sentence)
+
```

Comment:
I think we would consider this a Robocup hack. Not sure if it is smart to add this to the code. 
On the one hand Hero does not get stronger. But it is dependent on the exact bottle of water being used. Furthermore there may be other items we need to exclude based on their weight and this implementation is not configurable.